src/index.html

```diff
@@ -7,8 +7,53 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo__link">
+        <img
+          class="logo"
+          src="./images/logo.png"
+          alt="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+              data-qa="hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Gadgets</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Tablets</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Photo</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Video</a>
+          </li>
+        </ul>
+      </nav>
```

Comment:
```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a class="nav__link is-active" href="#">Apple</a>
          </li>
          
          <li class="nav__item">
            <a class="nav__link" href="#">Samsung</a>
          </li>
          
          <li class="nav__item">
            <a class="nav__link" href="#">Smartphones</a>
          </li>
          
          <li class="nav__item">
            <a
              class="nav__link"
              href="#"
              data-qa="hover"
            >
              Laptops & Computers
            </a>
          </li>
          
          <li class="nav__item">
            <a class="nav__link" href="#">Gadgets</a>
          </li>
          
          <li class="nav__item">
            <a class="nav__link" href="#">Tablets</a>
          </li>
          
          <li class="nav__item">
            <a class="nav__link" href="#">Photo</a>
          </li>
          
          <li class="nav__item">
            <a class="nav__link" href="#">Video</a>
          </li>
        </ul>
      </nav>
```

pay attention to style guide
![image](https://github.com/mate-academy/layout_moyo-header/assets/108538921/77e2fd94-48e8-4494-9992-cfd8049e87a4)